force-app/main/default/lwc/mVPNomineeBasicInfo/mVPNomineeBasicInfo.js-meta.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>56.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__AppPage</target>
+        <target>lightning__HomePage</target>
+        <target>lightning__RecordPage</target>
+    </targets>
+    <targetConfigs>
+        <targetConfig targets="lightning__RecordPage">
+            <property name="fieldSet" type="String" label="Field Set"/>
```

Comment:
set Default values for all properties as displayed in the UI
https://docs.google.com/document/d/1lhAf0tB-qO0Ly37pFDcs8K3IXNya6b-D/edit